stdlib/ballerina-builtin/src/main/ballerina/ballerina/compression/natives.bal

```diff
@@ -0,0 +1,21 @@
+package ballerina.compression;
+
+@Description {value:"Function to decompress compressed bytes"}
+@Param {value:"content: Byte array of the compressed file"}
+@Param {value:"destDir: Destination directory to place the decompressed file"}
+public native function unzipBytes (blob content, string destDir);
+
+@Description {value:"Function to decompress a zipped file"}
+@Param {value:"content: Path of the zipped file"}
+@Param {value:"destDir: Destination directory to place the decompressed file"}
+public native function unzipFile (string dirPath, string destDir);
+
+@Description {value:"Function to compress a folder"}
+@Param {value:"dirPath: Path of the directory to be compressed"}
+@Param {value:"destDir: Destination directory to place the compressed file"}
+public native function zipFile (string dirPath, string destDir);
+
+@Description {value:"Function to compress bytes"}
+@Param {value:"dirPath: Path of the directory to be compressed"}
+@Return {value:"compressed bytes of the file"}
```

Comment:
It should be "Compressed bytes of the file"

Reply:
Fixed 5ea59f6